addons/events/.storybook/components/Logger.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import json from 'format-json';
+import PropTypes from 'prop-types';
+import EventEmiter from 'eventemitter3';
+
+import { EVENTS } from '../constants';
+
+const styles = {
+  wrapper: {
+    padding: 20,
+    fontFamily: 'Areal, sans-serif',
```

Comment:
Is this a typo for `Arial`? Might want to include `Helvetica` for mac users too.